app/views/spree/components/products/_shipment_items.html.erb

```diff
@@ -0,0 +1,22 @@
+<ul class="shipment-items">
+  <% products.each do |product| %>
```

Comment:
I think this variable name should match the file name and should be `shipment_items do |shipment_item|`. I'm also unsure that `components/products` is a good place for this. Maybe it deserves its own namespace since at the end it's not highly related to products.

Reply:
Makes sense, will fix soon. Thanks!

Reply:
Fixed the naming and moved the file to the checkout folder for the time being. Let me know if you have a better idea in mind.

Reply:
Just noticed that I need do move the css files.